app/domain/export/ics/events.rb

```diff
@@ -21,13 +21,27 @@ def generate_ical_from_event_dates(event)
       end
     end
 
+    def event_description(event)
```

Comment:
- Reduce the method length: **done** (it used to ok with the previous rubocop config)
- Replacement of double quotes with single quotes:  **done**
- Including url_helpers in the class: **done**. I had it that way because I felt that using it only once was not enough to include it.